mapadroid/worker/WorkerQuests.py

```diff
@@ -619,6 +650,9 @@ def _current_position_has_spinnable_stop(self, timestamp: float):
                 fort_type: int = fort.get("type", 0)
                 if fort_type == 0:
                     self._db_wrapper.delete_stop(latitude, longitude)
+                    # TODO: if you know what fort_type 0 means, please edit
+                    # this logline accordingly :)
+                    logger.warning("Can't spin the stop because of fort_type 0")
```

Comment:
fort type 0 are gyms - this is why the if line deletes the stop from the DB. This shouldn't actually happen anymore though, since PD blocks gyms opening up when they are clicked. Code should be left in here for those special rare cases tho.

TLDR; change text to "Tried to open a stop but found a gym instead"